August Office Add-ins community call (#1908)
pnp/blog: content/office-add-ins-community-call/2024-10-09/index.md

---
title: "Office Add-ins developer platform community call – October 9, 2024"
summary: "Topics include: Install JS add-ins together with main app/COM add-ins on Win32 by Skylar Pan, Senior Product Manager at Microsoft. Call hosted by Mingjia Liu, Product Manager2 at Microsoft. Recorded on October 9, 2024."
date: 2024-10-09
author: "Mingjia Liu"
githubname: MingjiaLiu1995
categories: ["Office add in developer community call"]
images:
- images/office-add-ins-community-call-2024-10-09.png
tags: ["Office Add-ins","Microsoft 365"]
type: "regular"
videos:
- https://youtu.be/7oDPawF58rg
draft: false
---

# Office Add-ins developer platform community call - October 9, 2024

## This month's agenda and presenters

> The call was hosted by [Mingjia Liu](https://www.linkedin.com/in/mingjia-liu-90a69a24a/), Product Manager2, Microsoft.

Reviewer: ```suggestion
The call was hosted by [Mingjia Liu](https://www.linkedin.com/in/mingjia-liu-90a69a24a/), Product Manager 2 at Microsoft.
```

> * **Install JS add-ins together with main app/COM add-ins on Win32** [Skylar Pan](https://www.linkedin.com/in/skylar-pan/), Senior Product Manager at Microsoft.

Reviewer: ```suggestion
* **Install JS add-ins together with main app/COM add-ins on Win32** [Skylar Pan](https://www.linkedin.com/in/skylar-pan/), Senior Program Manager at Microsoft.
```

    * [Word](https://learn.microsoft.com/office/dev/add-ins/quickstarts/word-quickstart)
    * [PowerPoint](https://learn.microsoft.com/office/dev/add-ins/quickstarts/powerpoint-quickstart)
* [Script Lab](https://aka.ms/getscriptlab)
* [Samples](https://aka.ms/officeaddinsamples)
* [Microsoft 365 Developer Program](https://aka.ms/M365devprogram)
* [Office Scripts](aka.ms/office-scripts-docs)
* [Technical questions about Office add-ins](https://aka.ms/office-addins-dev-questions)
* [Stack Overflow questions](https://stackoverflow.com). Use keywords **office-js**, **outlook-web-addins**, or **office-scripts**.
* [Github office-js issues](https://github.com/OfficeDev/office-js/issues)
* [Microsoft Tech Community – Submit feature requests](https://aka.ms/m365dev-suggestions)
* [Microsoft 365 Developer Program](https://aka.ms/M365devprogram)

## Stay connected

* See the full blog post for this call in the [Microsoft 365 platform community blog](https://aka.ms/m365pnp/blog)
* [X (formerly Twitter)](https://twitter.com/microsoft365dev)
* [Microsoft 365 Unified Sample gallery](https://aka.ms/community/samples)

> * [Microsoft 365 Platform Community in YouTube](https://aka.ms/community/videos)

Reviewer: ```suggestion
* [Microsoft 365 Platform Community on YouTube](https://aka.ms/community/videos)
```
